flask_dance/contrib/slack.py

```diff
@@ -61,14 +64,16 @@ def make_slack_blueprint(
     :returns: A :ref:`blueprint <flask:blueprints>` to attach to your Flask app.
     """
     scope = scope or ["identify", "chat:write:bot"]
+    subdomain = subdomain or ""
+    if subdomain is not "": subdomain += "."
```

Comment:
This would probably be clearer:
```suggestion
    if subdomain:
        hostname = "{subdomain}.slack.com".format(subdomain=subdomain)
    else:
        hostname = "slack.com"
```

And then use it below